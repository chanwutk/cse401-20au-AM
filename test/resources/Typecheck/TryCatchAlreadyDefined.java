class TryCatchAlreadyDefined {
  public static void main(String[] args) {
    System.out.println(new C1().run());
  }
}

class C1 {
  int a;
  public int run() {
    int b;
    try {

    } catch (RuntimeException a) {
      a = 0;
    }

    try {}
    catch (RuntimeException b) {
      b = 0;
    }
    return b;
  }
}
