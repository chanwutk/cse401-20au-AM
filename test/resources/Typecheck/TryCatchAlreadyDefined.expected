TryCatchAlreadyDefined.java:13: error: RuntimeException a is already defined in classC1
TryCatchAlreadyDefined.java:18: error: RuntimeException b is already defined in method run(...)
